fe2-android/app/src/main/java/com/github/dedis/popstellar/ui/detail/event/election/fragments/CastVoteFragment.java

```diff
@@ -63,7 +63,7 @@ public class CastVoteFragment extends Fragment {
                   mLaoDetailViewModel.getCurrentElection().getId());
           electionVotes.add(electionVote);
         }
-        mLaoDetailViewModel.sendVote(electionVotes);
+        mLaoDetailViewModel.sendVote(electionVotes, getParentFragmentManager());
```

Comment:
Isn't it suspect to give a Fragment manager as a variable to a "sendVote(...)" function ?

Reply:
That's because currently, `sendVote()` (and multiple other functions) don't have callbacks. The caller cannot know when the vote is sent and what the result was.
Everything is coded inside the function, including the change of fragment when the vote is successfully sent.

This is part of a plan that we have for the future: to migrate these kinds of functions such that they return a callback.
But this will be a huge change and a dedicated PR is in the work.